sections/newsletter.liquid

```diff
@@ -124,7 +124,7 @@
       "settings": [
         {
           "type": "richtext",
-          "id": "paragraph",
+          "id": "text",
```

Comment:
Similarly here, if the block limit is `1`, static name should suffice. What do you think?

Reply:
Yeah that's what I was thinking too. From what I understood, it might be useful when deciding which icons to use next to a block 🤔 
![](https://screenshot.click/21-08-p5rwl-wjtgs.png)